### PR TITLE
MC-10547: update api doc, idk what else I can add...

### PR DESCRIPTION
**Pull request by JamieWulala**
Idk what else I can add 😞

### CODE REVIEW

**Review by jasondias9 — COMMENTED** [review]
How have we been documenting the force delete? We should probably put some information about this

File: source/includes/administration/_environments.md [hunk 1]
```diff
@@ -482,6 +482,16 @@ curl -X DELETE "https://cloudmc_endpoint/v2/environments/[environment-id]/member
...
 
+```shell
+# Response body example
+```
+```json
+{
+  "taskId": "a82113b9-7f76-4354-ae27-b18b0204d50f",
+  "status": "PENDING"
+}
+```
```
Comment (pdube) [inline source/includes/administration/_environments.md:497]: This looks good. We could also mention above that it is an asynchronous API, so we could link the Tasks section. Should we update the resource endpoint above to `/v2/environments` ?

**Review by pdube — APPROVED** [review]

File: source/includes/administration/_environments.md (hunk 1 above)
Comment (JamieWulala) [reply source/includes/administration/_environments.md:497]: the end point is already v2, I added a detail description

**Review by jasondias9 — APPROVED** [review]
LGTM

**Review by TheGreatHegazi — APPROVED** [review]
LGTM!

**Pull request merged**